src/includes/attachment-init/android.mdx

```diff
@@ -0,0 +1,14 @@
+```java
```

Comment:
H: The Android bits when it's exactly the same, fallback to the Java ones, so we don't actually need to have the android.mdx here, it's exactly the same as Java and will pick up the java.mdx automatically.

Reply:
That's new to me. Didn't know that we fixed this now. Thanks for pointing it out.